shell.middle/pom.xml

```diff
@@ -18,7 +18,11 @@
 
     <dependencies>
         <!-- From here on, all plugins are inherited from the parent project. -->
-
+        <dependency>
+            <groupId>org.metaborg.spoofax</groupId>
+            <artifactId>shell.core</artifactId>
+            <version>0.1-SNAPSHOT</version>
```

Comment:
In that case, this dep should be placed above the `<!-- From here on, ... -->` line.